e2e/e2e_scripts_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+// +build e2e_scripts
+
+package e2e
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/sylabs/singularity/internal/pkg/buildcfg"
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+	"github.com/sylabs/singularity/pkg/stest"
+	useragent "github.com/sylabs/singularity/pkg/util/user-agent"
+
+	// custom builtins
+	_ "github.com/sylabs/singularity/e2e/scripts_builtins/command"
+	_ "github.com/sylabs/singularity/e2e/scripts_builtins/test"
+)
+
+var testScripts = []struct {
+	name string
+	path string
+}{
+	{"BUILD/BASIC", "build/basic.test"},
```

Comment:
As a part of our testing best practices (which we're working on), it would be good to be more verbose here with this struct.

Example: `{ name: "ACTIONS/ENV_PATH", path: "actions/env_path.test" },`